src/models/product.model.js

```diff
@@ -0,0 +1,111 @@
+const { AppDataSource } = require("./data-source");
+
+const productList = async (userId) => {
+  // , userId, page, sort, category
+  let input = 0;
+  if (userId) {
+    input = userId;
+  }
+  let query = `SELECT 
+    products.id AS id,
+    products.name AS name,
+    categories.name AS category,
+    JSON_ARRAYAGG(
+      JSON_OBJECT(
+          "id", product_images.order,
+          "url", product_images.url
+      ) 
+    ) AS productImg,
+    products.price - (products.price * (products.discount_rate / 100)) AS price,
+    products.price AS originalPrice,
+    products.discount_rate AS discountRate,
+    COUNT(likes.product_id) AS likeNumber,
+    COUNT(reviews.product_id) AS reviewNumber,
+    IF(DATEDIFF(NOW(),products.created_at) < 30, "true", "false") AS isNew,
+    IF(liked.user_id="1" || liked.user_id= 1, "true", "false") AS isLike,
```

Comment:
```suggestion
    IF(DATEDIFF(NOW(),products.created_at) < 30, 1, 0) AS isNew,
    IF(liked.user_id="1" || liked.user_id= 1, 1, 0) AS isLike,
```

"true" "false"를 1, 0으로 대체하여 아래에서  isLike와 isNew를 변환할 때 활용하면 좋겠습니다.